qobuz_importer.user.js

```diff
@@ -132,13 +185,28 @@ function insertLink(release) {
 
     let mbUI = $('<p class="musicbrainz_import">')
         .append(MBImport.buildFormHTML(parameters) + MBImport.buildSearchButton(release))
-        .hide();
-    mbUI.append($('<button id="isrcs" type="submit" title="Show list of ISRCs">Show ISRCs</button>'));
+        .hide()
+        .append($('<button id="isrcs" type="submit" title="Show list of ISRCs">Show ISRCs</button>'));
+    if (classical) {
+        let title = 'Release data was detected as classical. Click to parse as artist.';
+        if (!detect) title = 'Click to reparse as classical.';
+        mbUI.append($(`<button id="reparse" type="submit" title="${title}">Reparse</button>`));
+        mbUI.append(
+            $(
+                `<span><input id="force_aa" type="checkbox" title="Force album artist"${force_aa &&
+                    ' checked'}><label for="force_aa">Force album artist</label></span>`
```

Comment:
It'd be better if the checkbox and the label were on the same line (Firefox here).
![image](https://user-images.githubusercontent.com/151042/48915006-6ce3ef00-ee7d-11e8-9fff-49085c8d527e.png)

Also tooltips aren't very clear about what it changes imho.
If it is possible we should just propose this choice if it is actually changing something.

Do we really need the "Reparse" option ? I mean what about doing parsing once, and keep both cases, if they differ.

Looks promising :)